src/pages/DatasetCatalog.js

```diff
@@ -392,23 +393,23 @@ class DatasetCatalog extends Component {
               table({ className: 'table' }, [
                 thead({}, [
                   tr({}, [
+                    th({ isRendered: this.state.isAdmin, className: 'cell-size', style: { textAlign: 'center' } }, ['Actions']),
```

Comment:
The actions column is a bit out of alignment. This is from Chrome logged in as admin

![Screen Shot 2020-12-14 at 10 11 09 AM](https://user-images.githubusercontent.com/116679/102098165-ddf3f780-3df4-11eb-896c-0594f10643ae.png)

Reply:
This is part of current state. I'm unsure how to resolve and will create a separate ticket.